lib/readline.js

```diff
@@ -77,6 +80,8 @@ function Interface(input, output, completer, terminal) {
   this.isCompletionEnabled = true;
   this._sawKeyPress = false;
   this._previousKey = null;
+  this._enableBuffer = false;
```

Comment:
Can you please add a comment on what this does? Also, I would prefer if this was a `Symbol` instead.